src/components/Input.tsx

```diff
@@ -32,29 +34,19 @@ function getRandomType(types: Record<string, Type<unknown>>): string {
   return names[Math.floor(Math.random() * names.length)];
 }
 
-const workerInstance = worker();
-
 const DEFAULT_FORK = "phase0";
 
 class Input<T> extends React.Component<Props<T>, State> {
+  worker: Worker;
+  // TODO: not sure what type to put here
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  workerInstance: any;
```

Comment:
This should be typed
https://threads.js.org/usage#type-safe-workers
You might also call it something more meaningful